SU-SWS/sulgryphon-nextjs
# SUL23-70 | @jdwjdwjdw | Build and style the library branch pages

# READY FOR REVIEW

# Summary
- [SUL23-70](https://stanfordits.atlassian.net/browse/SUL23-70): FE-DEV | Build and style the branch pages

# Review By (Date)
- When convenient

# Steps to Test

1. Checkout branch or view [preview](https://deploy-preview-44--stanford-library.netlify.app/sample-content/david-rumsey-map-center)
2. Create library branch page
3. Confirm everything looks correct, including responsive
4. Review code ☕

[SUL23-70]: https://stanfordits.atlassian.net/browse/SUL23-70?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
